docs/JavaScript/README.md

```diff
@@ -23,57 +21,10 @@ In order to improve the clarity, quality and development time it is worth consid
 
 ## Style Guide
 
--   [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript/blob/master/README.md) is partially being followed in our code base.
+-   [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript/blob/master/README.md) is being followed in our code base.
 
 -   Code formatting is handled entirely by Prettier. Run `npm run prettify` from the root of the project to format all code. Code is also automatically formatted [pre-commit](https://www.atlassian.com/git/tutorials/git-hooks).
```

Comment:
Why do we use prettify and eslint. Can we just stick with eslint?

Reply:
we included the prettier rules in [deriv eslint](https://github.com/binary-com/eslint-config-deriv/).
once we upgrade the Typescript on deriv-app we can proceed with using it and remove the prettier config entirely

Reply:
Prettier is much faster when it comes to formatting - https://www.reddit.com/r/node/comments/w95n80/why_use_prettier_if_eslint_can_format/
Since we do code formatting in the pre-commit hook, the formatting has to be faster.

Reply:
Thanks @balakrishna-deriv. Prettier is faster because it's mainly for formatting documents, no? Therefore does it really do all the checks we want?

Anyway, hopefully in the future as we split out the monolith into different repo's, it will reduce the time eslint takes and we can choose one linting tool. But seems like it makes sense to keep it in for now.